3.analyze_data/nbconverted/make_volcano_predicted_probabilities.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Compare phenotypic expression among treatments with volcano plot visualizations
+# We compare phenotypic expression for each (phenotype, statistical test, model type, treatment type) combination
+
+# In[ ]:
```

Comment:
Make sure to have run the notebook for this analysis prior to converting to python file so that the code cells are ran in order.